packages/ra-core/src/auth/useCheckAuth.ts

```diff
@@ -44,12 +45,14 @@ export const useCheckAuth = (): CheckAuth => {
     const authProvider = useAuthProvider();
     const notify = useNotify();
     const logout = useLogout();
+    const basename = useBasename();
+    const loginUrl = (basename ?? '') + defaultAuthParams.loginUrl;
```

Comment:
```suggestion
    const loginUrl = `${basename ?? ''}${defaultAuthParams.loginUrl}`;
```

Just nitpicking, feel free to ignore